source/instructors/admin/integration/grading.rst

```diff
@@ -3,15 +3,23 @@
 
 .. _lti-grading:
 
-Grading
-=======
+LMS Grading
+============
 Codio passes grades back to the LMS as a percentage earned out of the available points designated in Codio. 
 
-By default, grades are passed when assignment is completed and Release Grades setting is enabled, but if you want to customise/configure this behavior you can set **CUSTOM_CODIO_SEND_GRADE** custom parameter and here is list of possible values (case insensitive):
+By default, grades are passed when assignment is completed and **Release Grades** setting is enabled, but if you want to customise/configure this behavior you can in the **custom fields parameter** in the integration. You can set **CUSTOM_CODIO_SEND_GRADE** custom parameter and here is list of possible values (case insensitive):
```

Comment:
customise/configure - please change from UK spelling to US spelling (customize) and we don't need both words - just customize is enough.

Also, please add "the"
You can set the **CUSTOM_CODIO_SEND_GRADE**